SignalServiceKit/src/Util/NSObject+OWS.h

```diff
@@ -8,9 +8,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - Logging
 
-+ (NSString *)logTag;
+@property (nonatomic) NSString *logTag;
 
-- (NSString *)logTag;
++ (NSString *)logTag;
```

Comment:
does this mean class methods in swift will still need to do `self.logTag()`

Reply:
Good q - this PR fixes the instance usages.